code/035/script.js

```diff
@@ -0,0 +1,15 @@
+function User(username) {
+    this.username = username;
+  
+    this.updateUsername = (newName) => {
+      this.username = newName;
+    };
+  }
+  
+  const user1 = new User('kirandash');
+  const user2 = new User('bgwebagency');
+  
+  user1.updateUsername('-website');
+  user2.updateUsername('-app');
+  
+  console.log(user1.username + user2.username);
```

Comment:
I only see the script.js file in this PR. Please push the changes for README file as well.

If you are facing any issues, feel free to reach me on discord for any help